data_diff/__main__.py

```diff
@@ -30,7 +30,7 @@
 @click.option("-c", "--columns", default=[], multiple=True, help="Names of extra columns to compare")
 @click.option("-l", "--limit", default=None, help="Maximum number of differences to find")
 @click.option("--bisection-factor", default=32, help="Segments per iteration")
-@click.option("--bisection-threshold", default=1024**2, help="Minimal bisection threshold")
+@click.option("--bisection-threshold", default=1024 * 16, help="Minimal bisection threshold")
```

Comment:
Could we reference a constant here instead so we don't duplicate the default across here and the Python API? 👍🏻

Reply:
Could we? Let me check if such a thing is possible ;)